session1/examples/generics.ts

```diff
@@ -0,0 +1,33 @@
+//Generic type variables
+function checkPasswordLength<T>(password: T) {
+    console.log(password.length);
+}
+
+//Generic types
+function testArray<T>(item: Array<T>) {
+    return item;
+}
+let furits = testArray<string>(["Apples"]);
+furits.push(100);
```

Comment:
1. Let's show the items in the `fruits` array at the end. 
2. Why would someone logically put a number in an array of `fruits`?
3. This line would throw an error because the expected type is array of generic and not plain generic. Please add an array of generic of some other datatype so that we can show how generics is actually working.